jetty-proxy/src/main/java/org/eclipse/jetty/proxy/AbstractProxyServlet.java

```diff
@@ -145,11 +145,14 @@ public void destroy()
     {
         try
         {
-            _client.stop();
+            if (_client != null)
+                _client.stop();
```

Comment:
Use:
```suggestion
            LifeCycle.stop(_client);
```
As it is more concise and also avoids any race conditions if _client changes value.

Reply:
@gregw 👍 Thank you for the review. Done.